yourgov/components/Staff/DataTable.tsx

```diff
@@ -33,12 +33,14 @@ const descriptionId = 'data-table-description';
 type DataTableProps = {
 	/** The id of the heading that describes the table */
 	headingId?: string;
+	/** On close of the action modals, this element will be focused, rather than the trigger element. */
```

Comment:
I considered using context, but a few props were easier

Reply:
Should this be called `pageAlertElement` since the type is `HTMLElement`?

Reply:
ah, yeah, probably

Reply:
Done